template.tpl

```diff
@@ -265,148 +226,163 @@ const isLoggingEnabled = determinateIsLoggingEnabled();
 const traceId = getRequestHeader('trace-id');
 const eventData = getAllEventData();
```

Comment:
add logs block to UI and update logs determinateIsLoggingEnabled as here https://github.com/stape-io/gads-offline-conversion-tag/blob/main/template.js#L533